05_get_stats_02_identifying_data.Rmd

```diff
@@ -19,56 +19,40 @@ Thus, although each of us generate data daily through the apps we use or through
 
-In this lesson, we're going to really focus on the second and third questions there. What data do you need to answer your data science question and what limitations do these data have? We'll focus on the details of how to get the data necessary to answer the question of interest. We'll walk through this process again using the [Trump Tweets blog post](http://varianceexplained.org/r/trump-tweets/) as an example.
+In this lesson, we're going to really focus on the second and third questions there. What data do you need to answer your data science question and what limitations do these data have? We'll focus on the details of how to get the data necessary to answer the question of interest. We'll walk through this process again using the [Dimiter Toshkov’s blog](http://re-design.dimiter.eu/?p=767) as an example.
 
 
-![Trump Tweets Blog Post](https://docs.google.com/presentation/d/1hin5y4jDZikoGLbIEnwnPhJybgD_FezWsh16QIu5C5o/export/png?id=1hin5y4jDZikoGLbIEnwnPhJybgD_FezWsh16QIu5C5o&pageid=g3ec882b221_0_155)
+![IMDB](https://docs.google.com/presentation/d/1hin5y4jDZikoGLbIEnwnPhJybgD_FezWsh16QIu5C5o/export/png?id=1hin5y4jDZikoGLbIEnwnPhJybgD_FezWsh16QIu5C5o&pageid=g3ec882b221_0_155)
 
 ### The Perfect Dataset: The Data We Want
 
 Once you determine your data science question of interest, it's helpful to **envision the perfect dataset**. This dataset likely doesn't exist in the real world, but our goal will be to get data that match this optimal dataset *as closely as possible*.
 
 Thus to answer the question
 
-> Do the tweets on the _@realDonaldTrump_ account differ between those posted using an iPhone vs. those posted using an Android? Are the Android tweets angrier and more negative?
+> What variables from IMDB predict the author's movie ratings?
 
 we would ideally want a dataset where each row is a different tweet with each row containing information on the following variables:
 
-* `date` - date the tweet was written
-* `time` - time the tweet went out
-* `os` - the operating system of where the tweet came from (Android or iPhone)
-* `author`- author of each tweet
-* `location` - where the author was when the tweet was written
-* `tweet` - what the tweet said
-* `censored` - whether or not the tweet was later deleted
-  * `censored_date` - date on which the tweet was deleted
-  * `censored_time` - time at which the tweet was deleted
-* `anger` - some measurement of how angry and negative the tweet is
-
-Additionally, in a perfect world, we'd have all the tweets ever tweeted from _@realDonaldTrump_.
+- TODO: Explaining the ideal variables in the dataset
```

Comment:
This is a bit tricky to imagine since the analysis is exploratory.

Reply:
I must have deleted too much when I slimmed down the document as I had already done this. Let me add it now

Reply:
I don't have the admin right to edit the file, but id you want to add these:

•	rating – our own rating for the movie
•	imdb – the IMDb movie rating
•	genre – the genre of the movie
•	director – Director of the movie
•	year – Year of release of the movie